.werft/observability/install-satellite.sh

```diff
@@ -24,14 +24,11 @@ if ! command -v envsubst; then
   go install github.com/a8m/envsubst/cmd/envsubst@latest
 fi
 
-obsDir="${SCRIPT_PATH}/observability"
-mkdir -p "${obsDir}"
-git clone https://roboquat:"$(cat /mnt/secrets/monitoring-satellite-preview-token/token)"@github.com/gitpod-io/observability.git "${obsDir}"
-cd "${obsDir}/installer"
+GOBIN=$(pwd) go install github.com/gitpod-io/observability/installer@main
```

Comment:
would it be possible to have the binary be named "observability-installer" just to avoid confusing it with the Gitpod Installer.

Alternatively `rm ./installer` after line 31 would also work